pysat/tests/test_utils_coords.py

```diff
@@ -119,10 +119,12 @@ def test_bad_lon_name_calc_solar_local_time(self):
                                          lon_name="not longitude",
                                          slt_name='slt')
 
-    def test_lon_broadcasting_calc_solar_local_time(self):
+    @pytest.mark.parametrize("name", ["testmodel", "testing2d",
+                                      "testing2d_xarray"])
+    def test_lon_broadcasting_calc_solar_local_time(self, name):
         """Test calc_solar_local_time with longitude coordinates."""
 
-        self.py_inst = pysat.Instrument(platform='pysat', name="testmodel")
+        self.py_inst = pysat.Instrument(platform='pysat', name=name)
```

Comment:
Ran this manually on my end for additional confirmation. Also made a plot for the xarray case. Min and max check out as well.

![test_plot](https://user-images.githubusercontent.com/3595641/110664005-a6df7900-818c-11eb-8e22-245bcc86c0d4.png)